src/pages/pricing.tsx

```diff
@@ -77,13 +78,13 @@ const PricingPage: FunctionComponent = () => (
     >
         <div className="text-dark">
             <div className="mt-2">
-                <ContentSection className="hero-section text-center py-5">
-                    <h1 className="display-2 font-weight-bold">Sourcegraph Pricing</h1>
-                    <h4 className="font-weight-normal pt-2">Self-hosted code intelligence platform</h4>
+                <ContentSection className="py-5 text-center hero-section">
+                    <h1>Sourcegraph Pricing</h1>
+                    <h4 className="pt-2">Self-hosted code intelligence platform</h4>
```

Comment:
These are the kinds of classes we want to avoid adding to headings because the DLS has defined guidelines on what headings to use. Each heading already has pre-defined font weights. I've defined them all in globals.css for our convenience so we can write a suitable heading without having to add an extra class.

However, we may come across some one-off cases where we need to reference font weights or other typography classes for example it's tailored to a specific design template, but for most cases, we should just swap to the appropriate heading. In this case, it should be an h3 since the DLS references it as a subheading. Fixed in [2b5eeeb](https://github.com/sourcegraph/about/pull/5654/commits/2b5eeebe7a4becd2d8ea0909da55b679b967edc3).

Reply:
Perfect! Yea I was getting at the typography doesn't match in this instance and a few of the others I flagged. Sometimes the Figma is contradictory in it's hero headers font weight & size. Will definitely reinforce not adding typography utility classes to headers in most scenarios 🙌🏻